src/Umbraco.Core/Deploy/IDataTypeConfigurationConnector.cs

```diff
@@ -1,14 +1,12 @@
-using System.Collections.Generic;
-using System.Diagnostics.CodeAnalysis;
+﻿using System.Collections.Generic;
 
 namespace Umbraco.Core.Deploy
 {
     /// <summary>
-    /// Defines methods that can convert a data type configurations to / from an environment-agnostic string.
+    /// Defines methods that can convert a configuration to / from an environment-agnostic string.
     /// </summary>
-    /// <remarks>Configurations may contain values such as content identifiers, that would be local
+    /// <remarks>Configuration may contain values such as content identifiers, that would be local
     /// to one environment, and need to be converted in order to be deployed.</remarks>
-    [SuppressMessage("ReSharper", "UnusedMember.Global", Justification = "This is actual only used by Deploy, but we dont want third parties to have references on deploy, thats why this interface is part of core.")]
```

Comment:
Please don't remove this. Otherwize the interface appears as not in use.

Reply:
Fixed